synthesizer/process/src/lib.rs

```diff
@@ -170,9 +276,24 @@ impl<N: Network> Process<N> {
         lap!(timer, "Load circuit keys");
 
         // Add the stack to the process.
-        process.add_stack(stack);
+        process.credits = Some(Arc::new(stack));
+
+        finish!(timer, "Process::load_from_storage");
+        // Return the process.
+        Ok(process)
+    }
+
+    /// Initializes a new process without creating the 'credits.aleo' program.
+    #[inline]
+    pub fn load_no_storage() -> Result<Self> {
```

Comment:
is the difference between the current `setup` function and this one the addition of `credits.aleo`? in that case one should ideally be redefined using the other

Reply:
Good one f812a4b5f